src/ai/ai.service.ts

```
@@ -0,0 +1,173 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Chess } from 'chess.js';
+import { Inject, forwardRef } from '@nestjs/common';
+import { GameService } from '../modules/game/game.service';
+import { GameStatus } from 'src/shared/enum/game.enum';
+import { Game } from 'src/schema/game.schema';
```

ESLint

[error] 'Game' is defined but never used.

(@typescript-eslint/no-unused-vars)